ItoR.py

```diff
@@ -9,8 +9,11 @@
         int1 = input()
 """
 def convert(int1):
+    if type(int1) != int:
+        return 'The input must be an integer'
+    if int1 > 3999 or int1 < 1:
+        return 'Number must be an integer between 1 and 3999'
     res = ''
-    int1 = int(int1)
     romancollection = ["M","CM","D","CD","C","XC","L","XL","X","IX","V","IV","I"]

```

Comment:
Good, also remember that constant data usually goes outside the function implementation, e.g. at the beginning of your file. I regraded.